recipe/meta.yaml

```diff
@@ -22,13 +22,14 @@ requirements:
     - ipykernel
     - traitlets
     - notebook
+    - widgetsnbextension
```

Comment:
Does it need to be pinned for some reason?

This will automatically get the latest version, which after your PR ( https://github.com/conda-forge/widgetsnbextension-feedstock/pull/5 ) is merged will be 1.2.2.